espflash/src/flasher/mod.rs

```diff
@@ -463,8 +463,8 @@ impl DeviceInfo {
     pub fn rom(&self) -> Option<Vec<u8>> {
         match self.chip {
             Chip::Esp32 => {
-                if let Some((major, _)) = self.revision {
-                    if major >= 3 {
+                if let Some((_, minor)) = self.revision {
```

Comment:
these changes are needed after fixing the efuse reading (in a previous PR) - I just noticed it when working on this